src/bin/stratisd.rs

```diff
@@ -61,6 +52,17 @@ fn write_err(err: StratisError) -> StratisResult<()> {
     Ok(())
 }
 
+pub static mut DEBUG: bool = false;
```

Comment:
If ```dbgp!``` was moved, rather than copied, shouldn't there be a corresponding ```-``` entry in this commit?